example/analysis_options.yaml

```
@@ -0,0 +1,14 @@
+
+include: package:very_good_analysis/analysis_options.yaml
+
+
+language:
+    strict-casts: true
+    strict-inference: true
+    strict-raw-types: true
+
+linter:
+  rules:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)